### ISSUE TITLE
Fix message format for SA1003

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1004]
The message format for SA1003 does not make sense. The following shows the result of 1.0.0-alpha011 running on commit deb09207e2fd303325083a2e1c4ca178ecdbf05e.

![image](https://cloud.githubusercontent.com/assets/1408396/8777251/85e6b372-2ebc-11e5-8081-aba33d1efc44.png)

**Comment by sharwell** [comment]
This issue occurs when SA1003 is reported for a space following a cast:

```
(Type) value
```

See commit 8f30d8f68df0833e524fc9614843f2a8faa5656f for examples of violations which trigger the poorly-formatted warning.

**Comment by sharwell** [comment]
I just realized that the problem is the types used in the cast expressions for this project are the same types as appear in the syntax tree. When the wording of this diagnostic says "Operator '(EventDeclarationSyntax)' must...", it is referring to an actual cast to `EventDeclarationSyntax` in the source code.
